main.go

```diff
@@ -590,36 +619,72 @@ func checkRCTagsAndVersions(c *cli.Context) {
 }
 
 func enforceLifecycle(c *cli.Context) {
-
 	// Initialize dependencies with branch-version, current chart and debug mode
+	logrus.Info("Initializing dependencies for enforce-lifecycle")
 	repoRoot := getRepoRoot()
 	rootFs := filesystem.GetFilesystem(repoRoot)
-
 	lifeCycleDep, err := lifecycle.InitDependencies(rootFs, c.String("branch-version"), CurrentChart, DebugMode)
 	if err != nil {
-		logrus.Fatalf("encountered error while initializing dependencies for lifecycle-assets-clean: %s", err)
+		logrus.Fatalf("encountered error while initializing dependencies: %s", err)
 	}
 
 	// Apply versioning rules
+	logrus.Info("Starting to enforce lifecycle rules for assets")
 	err = lifeCycleDep.ApplyRules(CurrentChart, DebugMode)
 	if err != nil {
 		logrus.Fatalf("Failed to apply versioning rules for lifecycle-assets-clean: %s", err)
 	}
 }
 
 func lifecycleStatus(c *cli.Context) {
-
 	// Initialize dependencies with branch-version and current chart
+	logrus.Info("Initializing dependencies for lifecycle-status")
 	rootFs := filesystem.GetFilesystem(getRepoRoot())
-
 	lifeCycleDep, err := lifecycle.InitDependencies(rootFs, c.String("branch-version"), CurrentChart, false)
 	if err != nil {
-		logrus.Fatalf("encountered error while initializing dependencies for lifecycle-assets-clean: %s", err)
+		logrus.Fatalf("encountered error while initializing dependencies: %s", err)
 	}
 
 	// Execute lifecycle status check and save the logs
-	err = lifeCycleDep.CheckLifecycleStatusAndSave(CurrentChart)
+	logrus.Info("Checking lifecycle status and saving logs")
+	_, err = lifeCycleDep.CheckLifecycleStatusAndSave(CurrentChart)
 	if err != nil {
 		logrus.Fatalf("Failed to check lifecycle status: %s", err)
 	}
 }
+
+func autoForwardPort(c *cli.Context) {
+
+	if ForkURL == "" {
+		logrus.Fatalf("FORK environment variable must be set to run auto-forward-port")
+	}
+
+	// Initialize dependencies with branch-version and current chart
+	logrus.Info("Initializing dependencies for auto-forward-port")
+	rootFs := filesystem.GetFilesystem(getRepoRoot())
+	lifeCycleDep, err := lifecycle.InitDependencies(rootFs, c.String("branch-version"), CurrentChart, false)
+	if err != nil {
+		logrus.Fatalf("encountered error while initializing dependencies: %v", err)
```

Comment:
Should we also be returning here too?

Reply:
The entire project was made following this standard of `fatal erroring` when an error reaches the function called by the CLI. 

I need to find out which architecture this CLI app is following. I did not spend time on it; I just followed through with what was in place. 

That being said, this function does not allow for returning errors. 
I need to spend time understanding this. 
But I would prefer to do that after the 2.9.0 release cycle.

Reply:
Ah, I see it now.